### PR TITLE
feat: remove kakarotup and associated usage from README

### LINKED ISSUE #138: feat: replace kakarotup in all repo by unwrapped equivalent
As a general rule, we are in favor of minimal abstractions when possible (no magic as would others say).

Here, kakarotup does not add much value, it'd rather stick with the usual cargo commands for our use-case.

Let's remove all instances of `sh kakarotup` and mentions of it in the readme and replace them by the natural usage of runs: `cargo build`, `git pull` etc. 

1. Check what kakarotup does
2. Replace its mentions in readme
3. Replace its mentions in makefile
4. Check that the requirements for the repo are clear (rust, git, etc.)

### PR DESCRIPTION
**Pull request by bajpai244**
This PR removes Kakarotup from the project as discussed in #138, the script brings a lot of complexity which might be overkill for now since we are already offering a docker image.

The current README has been updated to use `make build`, which enables building from source using cargo.

Closes #138

# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe):

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,29 +76,30 @@ Here is a high level overview of the architecture of Kakarot RPC.
...
 - Make
 
-### Installation
+## Installation
 
-#### make install
+### build from source
 
-To install with `make install` ( this requires nightly rustup):
+To build the project from source do `make build` ( this requires nightly rustup):
```
Comment (Eikix) [inline README.md:96]:
```suggestion
To build the project from source do `make build` (this requires [nightly rustup](https://rust-lang.github.io/rustup/concepts/channels.html)):
```

File: Makefile [hunk 2]
```diff
@@ -13,7 +8,7 @@ devnet: poetry-install
 
 # build
 build:
-	cargo build --all --release
+	cargo build -q --all --release
```
Comment (Eikix) [inline Makefile:11]: why add the --quiet flag?

**Review by Eikix — COMMENTED** [review]
awesome!

File: Makefile (hunk 2 above)
Comment (bajpai244) [reply Makefile:11]: It was there in kakrotup, so I updated my implementation accordingly, I can remove it, lemme know if you think it should be there and I will update.

**Comment by bajpai244** [comment]
@Eikix I have fixed the typo, so I guess running **spell check** again should pass.

File: Makefile (hunk 2 above)
Comment (Eikix) [reply Makefile:11]: you can remove it:)

Comment (bajpai244) [reply Makefile]: done ser 🚀

**Review by Eikix — APPROVED** [review]
lgtm ✅🌋🫡🫡

**Pull request merged**